torch/csrc/jit/passes/frozen_ops_to_mkldnn.cpp

```diff
@@ -380,9 +380,17 @@ class MKLDNNSubgraphSlicer {
     }
     // unary ops we dont need to prove anything else than
     // the input is mkldnn supported
-    if (n->kind() == aten::relu || n->kind() == aten::sigmoid) {
-      return true;
+    switch (n->kind()) {
+      case aten::relu:
+      case aten::sigmoid:
+      // TODO: max_pool on mkldnn can be slower than in eager. ideally, we'd
```

Comment:
do we have data how often max_pool is slower than eager and by how much?

Reply:
i can regenerate it